src/lib/components/Cart/CartDetails.svelte

```diff
@@ -92,7 +98,7 @@
 				{$t('cartdetails.tab.confirmation')}
 			</div>
 			<!-- close button-->
-			<div class="ml-auto pb-1 pr-14 duration-100 z-50 hover:scale-110">
+			<div class="ml-auto pb-1 pr-14 duration-100 z-50 hover:scale-110 -mt-8">
```

Comment:
I understand why you added the margin. It looks better when there is some space between the line and the icon. However, this would lead to the close button and the shopping cart button not be aligned anymore, which is from a visual standpoint more annoying (at least in my opinion).
What do you think?